cockatrice/src/main.cpp

```diff
@@ -97,6 +100,19 @@ bool settingsValid()
         !settingsCache->getPicsPath().isEmpty();
 }
 
+void generateClientID()
```

Comment:
Would you prefer the value be returned in a variable and then the setvalue function called to store it in the settings cache or do you care if I simply make it all a single line without requiring a variable to be defined?